package.json

```diff
@@ -1,9 +1,9 @@
 {
+  "type": "module",
   "name": "micro-credentials-platform-back",
-  "version": "0.1.0",
+  "version": "1.0.0",
```

Comment:
We're still in the initial development phase so let's make this version `0.2.0` (see [semver.org](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase)).